docs/collections/catalog/prefect-kubernetes.yaml

```diff
@@ -0,0 +1,6 @@
+collectionName: prefect-kubernetes
+author: Prefect
+authorUrl: https://prefect.io
+documentation: https://prefecthq.github.io/prefect-kubernetes/
```

Comment:
I get a 404 on this link

Reply:
I deployed the docs. I need to update the styling, but they should be available now.